spec/classes/init_spec.rb

```diff
@@ -77,6 +77,21 @@
         it { is_expected.not_to contain_class('pki')}
         it { is_expected.to create_file('/etc/pki/simp_apps/grafana/x509')}
       end
+
+
+      context 'when ldap is false' do
+        let(:params) { {:ldap => false} }
+        it_behaves_like 'a structured module'
+        it { is_expected.to contain_class('grafana').with_ldap_cfg( {} ) }
+      end
+
+
+#      context 'when ldap is true' do
+#        let(:params) { {:ldap => true} }
+#        it_behaves_like 'a structured module'
+#        it { is_expected.to contain_class('grafana').with_ldap_cfg( ) }
```

Comment:
it should, but the class was having problems inside the simp_openldap module. which is why we commented it out for now, and it was a bit inconsistent.